### ISSUE TITLE
S.O.S crashing

### ISSUE THREAD

**Issue by tvk1372** [Electrical-Age/ElectricalAge#114]
I have a big big big.. problem,I use your new lamp socket and put a 200v wire to the back of it  and I dont know how, but it kills my map. :(
I cant open it, when I wanted to open it it always says after 1 sec shutting down Internal server ! :(:(:(
And I dont have a working savefile of it!
Please Help how do I fix this crash?
I am working on this map months :( 
Crash report:
---- Minecraft Crash Report ----
// Quite honestly, I wouldn't worry myself about that.

Time: 2014.08.19. 23:56
Description: Rendering Block Entity

java.lang.NullPointerException: Rendering Block Entity
    at mods.eln.cable.CableRender.connectionType(CableRender.java:282)
    at mods.eln.node.six.SixNodeElementRender.draw(SixNodeElementRender.java:96)
    at mods.eln.sixnode.lampsocket.LampSocketRender.draw(LampSocketRender.java:82)
    at mods.eln.node.six.SixNodeRender.func_147500_a(SixNodeRender.java:56)
    at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147549_a(SourceFile:100)
    at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147544_a(SourceFile:92)
    at net.minecraft.client.renderer.RenderGlobal.func_147589_a(RenderGlobal.java:764)
    at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1538)
    at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1298)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:945)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:835)
    at net.minecraft.client.main.Main.main(SourceFile:103)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at mods.eln.cable.CableRender.connectionType(CableRender.java:282)
    at mods.eln.node.six.SixNodeElementRender.draw(SixNodeElementRender.java:96)
    at mods.eln.sixnode.lampsocket.LampSocketRender.draw(LampSocketRender.java:82)
    at mods.eln.node.six.SixNodeRender.func_147500_a(SixNodeRender.java:56)

-- Block Entity Details --
Details:
    Name: SixNodeEntity // mods.eln.node.six.SixNodeEntity
    Block type: ID #460 (tile.NodeBlock // mods.eln.node.six.SixNodeBlock)
    Block data value: 7 / 0x7 / 0b0111
    Block location: World: (-2,231,-38), Chunk: (at 14,14,10 in -1,-3; contains blocks -16,0,-48 to -1,255,-33), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
    Actual block type: ID #460 (tile.NodeBlock // mods.eln.node.six.SixNodeBlock)
    Actual block data value: 7 / 0x7 / 0b0111
Stacktrace:
    at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147549_a(SourceFile:100)
    at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147544_a(SourceFile:92)
    at net.minecraft.client.renderer.RenderGlobal.func_147589_a(RenderGlobal.java:764)
    at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1538)

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityClientPlayerMP['goku'/23, l='MpServer', x=-0,07, y=230,62, z=-34,57]]
    Chunk stats: MultiplayerChunkCache: 25, 34
    Level seed: 0
    Level generator: ID 01 - flat, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: World: (0,4,0), Chunk: (at 0,0,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
    Level time: 1370712 game time, 2836509 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 4 total; [EntityItem['item.tile.clayHardenedStained.silver'/5, l='MpServer', x=-1,88, y=229,13, z=-36,88], EntityChicken['Csirke'/8, l='MpServer', x=43,38, y=237,00, z=-37,91], EntityClientPlayerMP['goku'/23, l='MpServer', x=-0,07, y=230,62, z=-34,57], EntityItem['item.item.egg'/9, l='MpServer', x=41,34, y=237,13, z=-36,91]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Integrated singleplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:368)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2383)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:856)
    at net.minecraft.client.main.Main.main(SourceFile:103)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.7.2
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.7.0_67, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 510985264 bytes (487 MB) / 1111490560 bytes (1060 MB) up to 3817865216 bytes (3641 MB)
    JVM Flags: 2 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G
    AABB Pool Size: 39 (2184 bytes; 0 MB) allocated, 39 (2184 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.03 FML v7.2.217.1147 Minecraft Forge 10.12.2.1147 Optifine OptiFine_1.7.2_HD_D3 17 mods loaded, 17 mods active
    mcp{9.03} [Minecraft Coder Pack](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    FML{7.2.217.1147} [Forge Mod Loader](forge-1.7.2-10.12.2.1147.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Forge{10.12.2.1147} [Minecraft Forge](forge-1.7.2-10.12.2.1147.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    TooManyItems{1.7.2} [TooManyItems](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    malisiscore{1.7.2-0.8} [Malisis Core](malisiscore-1.7.2-0.8.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    CarpentersBlocks{3.2.5} [Carpenter's Blocks](Carpenter's Blocks v3.2.5 [DEV R3] - MC 1.7.2.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    SlimevoidLib{2.0.4.6} [Slimevoid Library](SlimevoidLibrary-2.0.4.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    DynamicTransport{0.1.1.0} [Dynamic Transport](DynamicTransport-0.1.1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Eln{39} [](ElectricalAge_BETA-1.9_r39.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    examplemod{1.0} [examplemod](ExtraDoors-1.7.2-1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    extradoors{1.5} [Extra Doors](ExtraDoors-1.7.2-1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    guilib{1.0.7.2} [GUI Library](Forge_GuiLibv1.0.7.2.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ImmibisCore{58.1.4} [Immibis Core](Immibis-Core-1.7.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    malisisdoors{1.7.2-0.9.1} [Malisis' Doors](malisisdoors-1.7.2-0.9.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    cfm{3.3.2} [§6MrCrayfish's Furniture Mod](MrCrayfishFurnitureModv3.3.2%281.7.2%29.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    RedLogic{58.1.3} [RedLogic](RedLogic-Mod-1.7.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MSC2{1.2.1} [Mob Spawn Controls 2](Forge_MSC2v1.2.1.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Launched Version: 1.7.2-Forge10.12.2.1147
    LWJGL: 2.9.0
    OpenGL: AMD Radeon R9 200 Series GL version 4.3.12618 Compatibility Profile Context 13.251.0.0, ATI Technologies Inc.
    Is Modded: Definitely; Client brand changed to 'fml,forge'
    Type: Client (map_client.txt)
    Resource Packs: [LIFE 128x (Vers. 80).zip]
    Current Language: Magyar (HU)
    Profiler Position: N/A (disabled)
    Vec3 Pool Size: 46 (2576 bytes; 0 MB) allocated, 46 (2576 bytes; 0 MB) used
    Anisotropic Filtering: Off (1)

**Comment by tvk1372** [comment]
Uhh I fixed it,I enter to the map with r37 release and its ok and I save it and I can continue my map with r39.But I dont know why it crash

**Comment by Dolu1990** [comment]
Damned XD
I fix that this affternoon and release r40 
Thanks for issue :)

**Comment by JPitman97** [comment]
Thanks for the issue :)

**Comment by Dolu1990** [comment]
I don't have successfuly reproduct it, Coud you send me your map ? With it i can verify that all is fixed :)

**Comment by tvk1372** [comment]
Hi!
I am not home at my computer,but I have the screenshot of the "damaged" block's in my laptop.
There are some carpenter blocks at the cables.
I marked the crash maker block with an X 
I put a 200v cable to the "X" marked place and that crashes my map 
This is the  floor -1
![error crash](https://cloud.githubusercontent.com/assets/8401485/3981049/02b59d6e-286a-11e4-9096-6c9c51d82d68.png)
And that  is above it this is the floor 0
There are also carpenter block's,but I think the cables doesn't touch the carpenter block.
![2014-08-20_15 05 01](https://cloud.githubusercontent.com/assets/8401485/3981193/58ac1ce2-286b-11e4-8262-097716c16d29.png)
And that is the hole I made on the  top of the roof of it
![2014-08-20_15 05 20](https://cloud.githubusercontent.com/assets/8401485/3981235/ddc7423a-286b-11e4-88f8-8e4714b7c246.png)
If you want my map to understand it better I can send you later when I go home.
